expenses/templates/add_payment.html

```diff
@@ -19,7 +18,7 @@
 <div class="container-fluid display-table">
     <div class="row display-table-row">
         <div class="col-md-2 col-sm-1 hidden-xs display-table-cell v-align box" id="navigation">
-            <div class="navi">
+            <section class="navi">
```

Comment:
Still incorrect tag.
```suggestion
            <nav class="navi">
```